### ISSUE TITLE
Omit disabled profile options from profile output

### ISSUE THREAD

**Issue by ashwinassysh** [capitalone/DataProfiler#488]
**Is your feature request related to a problem? Please describe.**
Currently the profile output provides all the statistics in the JSON output even if the user has disabled several statistics for specific data types.

**Describe the outcome you'd like:**
In the new output from the profiler the statistics which have been disabled should not be part of the profiler output JSON. There can be an attribute which can be set to True or False to control this feature, if in case someone needs it.

**Additional context:**

**Comment by taylorfturner** [comment]
Recreated the issue with the below: 
```
data = dp.Data(os.path.join(data_folder, "csv/diamonds.csv"))

profile_options = dp.ProfilerOptions()

# Setting multiple options via set
profile_options.set({ "data_type_representation.is_enabled": False, "row_count.is_enabled": False})

# Set options via directly setting them
profile_options.structured_options.data_labeler.max_sample_size = 1000

profile = dp.Profiler(data, options=profile_options)
report  = profile.report(report_options={"output_format":"compact"})

# Print the report
print(json.dumps(report, indent=4))
```

Output does include `row_count` and `data_type_representation`, when they shouldn't be included.

**Comment by JGSweets** [comment]
The original intent was to be a consistent output which is why they aren't removed. However, there can be a report_option added to address this.

**Comment by taylorfturner** [comment]
@ashwinassysh this is resolved in release `0.7.9`. Thanks to @micdavis for all the hard work!
